### PR TITLE
Feature/24941 make installable

### PR DESCRIPTION
**Pull request by jgj-magenta-aps**
This should make module able to be installed as a program using our salt stack.

### CODE REVIEW

**Review by istherepie — CHANGES_REQUESTED** [review]
The settings.py file is currently placed outside the scope of the package while it is specified that the file must exist in the path, is there any reason the file has not been included in the package ?

Additionally a flake8 check reveals cosmetic issues with the unit tests (line length and whitespaces).

**Comment by jgj-magenta-aps** [comment]
I will look into it today. 
I had envisioned the package as installed like an application, given that it has it's own cache - since it is before 'integrationdata', but there is actually no reason for that - it can be transformed into a package, except for the requirements.txt, which has to stay outside.

File: mox_cpr_delta_mo/__main__.py [hunk 1]
```diff
@@ -21,8 +21,9 @@
     cpr_get_all_subscribed,
 )
 
-from settings import (
+from .settings import (
```
Comment (istherepie) [inline mox_cpr_delta_mo/__main__.py:24]: It is preferable to use explicit import.

**Review by istherepie — CHANGES_REQUESTED** [review]
I am unable to run the unit tests as these depend on a configuration file - it may be beneficial to include either defaults/fallback values or a dummy configuration file in order to run the tests.

**Review by istherepie — APPROVED** [review]

**Pull request closed without merging**